testing/python/fixtures.py

```diff
@@ -1996,17 +2037,17 @@ def test_2(self):
                     pass
         """
         )
-        confcut = f"--confcutdir={testdir.tmpdir}"
-        reprec = testdir.inline_run("-v", "-s", confcut)
+        confcut = f"--confcutdir={pytester.path}"
+        reprec = pytester.inline_run("-v", "-s", confcut)
```

Comment:
Just to be a bit safer:

```suggestion
        reprec = pytester.inline_run("-v", "-s", "--confcutdir", str(pytester.path))
```

Reply:
changed in #8122, rebased